### ISSUE TITLE
Chart stuck in loop when number of facet terms exceeds default limit

### ISSUE THREAD

**Issue by chrisala** [AtlasOfLivingAustralia/biocollect#1323]
To reproduce:
1) Open coralwatch: https://biocollect-test.ala.org.au/coralwatch/bioActivity/allRecords?hub=coralwatch
2) Open javascript console / network tab

Expected:
Page initialises and completes

Actual:
Stuck in loop requesting extra facet info for the "Average colour code (record level)" facet
![image](https://user-images.githubusercontent.com/8001629/124374291-67114400-dcdd-11eb-885f-587a1405d036.png)

**Comment by chrisala** [comment]
The trigger for this is the number of terms returned for the average colour code facet is > the default max.

**Comment by sat01a** [comment]
Thanks Chris

**Comment by cofiem** [comment]
Yes, thanks, I can see why this is occurring. It is a bug in the graphs functionality. Please assign this issue to me.
